ui_testing/testcases/base_test.py

```diff
@@ -49,7 +51,10 @@ def fix_data_format(self, data_list):
         tmp = []
         for item in data_list:
             if len(str(item)) > 0:
-                if re.search(r'\d{2}.\d{2}.\d{4}', str(item)):
+                if re.search(r'\d{2}.\d{2}.\d{4},\s\d{1,2}:\d{1,2}\s(A|P)M', str(item)):                    
+                    tmp.append(datetime.datetime.strptime(item, '%d.%m.%Y, %H:%M %p'))
+                elif re.search(r'\d{2}.\d{2}.\d{4}', str(item)):
+                    # item = item.split(',')[0] # if the string have time, remove the time
```

Comment:
this gonna affect other test cases, Are u sure the test case using it is passing? if not, remove this change and create your own method

Reply:
@0xIslamTaha  No, it won't affect any other test cases